public/app/features/explore/Explore.tsx

```diff
@@ -514,7 +514,9 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
                             <ErrorBoundaryAlert>{this.renderFlameGraphPanel()}</ErrorBoundaryAlert>
                           )}
                           {showTrace && <ErrorBoundaryAlert>{this.renderTraceViewPanel()}</ErrorBoundaryAlert>}
-                          {showLogsSample && <ErrorBoundaryAlert>{this.renderLogsSamplePanel()}</ErrorBoundaryAlert>}
+                          {Boolean(config.featureToggles.logsSampleInExplore) && showLogsSample && (
```

Comment:
One nit: `Boolean` shouldn't be needed.

Reply:
Fixed in https://github.com/grafana/grafana/pull/62291/commits/ce650bbd74ef5752b2a4869ed189e33f7949abba